app/assets/stylesheets/common/base/login-signup-page.scss

```diff
@@ -156,6 +160,28 @@ body.signup-page {
       font-size: var(--font-down-1);
     }
 
+    .more-info {
+      opacity: 0;
+      max-height: 0;
+      transition: opacity 0.25s, max-height 0.25s;
+    }
+
+    input:focus + label + span.more-info {
+      opacity: 1;
+      max-height: 50px;
+    }
```

Comment:
> I had the impression that we wanted to avoid relying on animations to prevent layout shifts in the form and preferred to follow the FormKit specification, but I couldn't find the post or topic where I got that idea from, so I'm not sure.

I think for most if not all of Discourse core this should be a goal we strive for. That being said, I do feel the login / signup page is a bit on its own and warrants this diversion. I am not opposed to trying something different to blend the two styles.